salt/netapi/rest_tornado/saltnado.py

```diff
@@ -1100,13 +1100,15 @@ def _disbatch_runner(self, chunk):
         '''
         Disbatch runner client commands
         '''
+        full_return = chunk.pop('full_return', False)
```

Comment:
since in https://github.com/saltstack/salt/blob/2018.3/salt/netapi/rest_tornado/saltnado.py#L435
runner is linked to RunnerClient.cmd_async, which does not support full_return, we may just remove it from chunk:

chunk = {k: v for k, v in chunk.items() if k != 'full_return'}

I'm not clear yet as where it got into chunk. Maybe we can control it better at source